src/skeleplex/app/_curate.py

```diff
@@ -1,12 +1,30 @@
+import numpy as np
+import networkx as nx
+import sys
 from collections import deque
 from copy import deepcopy
 from typing import TYPE_CHECKING, Annotated, Any
-
+from skeleplex.graph.constants import (
+    EDGE_SPLINE_KEY,
+    NODE_COORDINATE_KEY,
+    GENERATION_KEY,
+)
+from magicgui import magicgui
+from skeleplex.visualize import EdgeColormap
 from skeleplex.graph.modify_graph import (
     connect_without_merging,
     delete_edge,
     merge_nodes,
+    split_edge
 )
+from PyQt6.QtWidgets import QWidget, QVBoxLayout, QLabel, QPushButton
+from PyQt6.QtGui import QPixmap
+from PyQt6.QtCore import QByteArray
```

Comment:
we want to import from qtpy. this allows us to use different Qt backeds (e.g., pyside, pyqt). For example: `from PyQt6.QtWidgets import QWidget` -> `from qtpy.QtWidgets import QWidget`

https://github.com/spyder-ide/qtpy